src/utils/scripts.ts

```diff
@@ -0,0 +1,35 @@
+import Address from '../models/address';
+import Network from '../models/network';
+import helpers from '../utils/helpers';
+import { unpackLen, unpackToInt } from '../utils/buffer';
+import _ from 'lodash';
+
+/**
+* Parse output script
+* We currently support only P2PKH
+* 
+* @param {Buffer} buff Output script
+* @param {Network} network Network to get the address first byte parameter
+*
+* @return {[number | null, Address]} Timelock and address from output script
+*/
+export const parseOutputScript = (buff: Buffer, network: Network): [number | null, Address] => {
```

Comment:
I prefer throwing error instead of returning null. Why should we return null?

Reply:
I agree. Throwing error is best.

Reply:
In fact, I'd like to add that I'm not sure how expensive it is to throw an exception in JavaScript. Some languages are really inefficient when it comes to exceptions while others are totally efficient. If it is efficient, I agree. Otherwise, I feel we should just return null because this method will be used together with others when we are trying to parse a given script.

Reply:
As discussed internally, I am keeping the throw Error but added this comment in the code.